src/main/java/agh/ics/oop/Animal.java

```diff
@@ -0,0 +1,30 @@
+package agh.ics.oop;
+
+public class Animal {
+    private Vector2d position = new Vector2d(2,2);
+    private MapDirection orientation = MapDirection.NORTH;
+    private final Vector2d upperRightBoundary = new Vector2d(4,4);
```

Comment:
fajnie, że Pan to wyciągnął do dedykowanego miejsca, tak docelowo tego typu wartości to stałe, warto jest oznaczyć modyfikatorami `static final` i zgodnie z konwencją pisze się je wtedy DUŻYMI_LITERAMI (będę o tym jeszcze mówił na zajęciach)